### ISSUE TITLE
Deprecated Workshop API

### ISSUE THREAD

**Issue by SupSuper** [greenheartgames/greenworks#237]
The current Workshop functions use the deprecated ISteamRemoteStorage functions, which has several limitations (limited to single files, limited metadata, manual downloads, etc).

It would be nice to have access to the newer ISteamUGC API and its capabilities, though it would probably require separate functions to maintain backwards-compatibility.

**Comment by kefniark** [comment]
Yes I just hit the same issue, I was trying to port existing code to greenworks, until I realized all the **ISteamUGC** APIs are missing.

Things described in this guide: https://partner.steamgames.com/doc/features/workshop/implementation
